src/TensorFlowNET.Core/Framework/importer.cs

```diff
@@ -121,7 +121,7 @@ private static void _ProcessNewOps(Graph graph)
 
             foreach(var input in input_map)
             {
-                throw new NotImplementedException("_PopulateTFImportGraphDefOptions");
+                c_api.TF_ImportGraphDefOptionsAddInputMapping(options.Handle, input.Key, 0, input.Value._as_tf_output());
```

Comment:
Is the `src_index` fixed value 0? It may cause issue.
Should call `_ParseTensorName` to parse:
```csharp
            foreach (var input in input_map)
            {
                var (src_name, src_index) = _ParseTensorName(input.Key);
                c_api.TF_ImportGraphDefOptionsAddInputMapping(options.Handle, src_name, src_index, input.Value._as_tf_output());
            }
```

Reply:
excellent, thank you for this. I updated the PR as you suggest.